consensus/src/main/java/org/apache/iotdb/consensus/ratis/Utils.java

```diff
@@ -265,4 +276,52 @@ public static void initRatisConfig(RaftProperties properties, RatisConfig config
     RaftServerConfigKeys.Rpc.setFirstElectionTimeoutMax(
         properties, config.getRpc().getFirstElectionTimeoutMax());
   }
+
+  /**
+   * Monitoring Ratis RaftLog total size. It will memorize the state of all files in the last update
+   * and calculates the diff incrementally each run.
+   */
+  static class RatisLogSizeMonitor {
```

Comment:
It feels like this class could be a separate tool class

Reply:
Moved to Utils package